Source/AtomicJS/Javascript/JSScene.cpp

```diff
@@ -24,6 +24,27 @@ static int Node_CreateJSComponent(duk_context* ctx)
     return 1;
 }
 
+static int Node_GetJSComponent(duk_context* ctx)
+{
+	StringHash classNameHash = duk_to_string(ctx, 0);
+	duk_push_this(ctx);
+	Node* node = js_to_class_instance<Node>(ctx, -1, 0);
+
+	PODVector<JSComponent*> jsComponents;
+	node->GetComponents<JSComponent>(jsComponents);
+
+	for (unsigned i = 0; i < jsComponents.Size(); i++)
+	{
+		if (StringHash(jsComponents[i]->GetClassName()) == classNameHash)
+		{
+			js_push_class_object_instance(ctx, jsComponents[i]);
+			break;
+		}
+	}
+
+	return 1;
+}
+
```

Comment:
If I have a component calls MyComponent, the last one added on  the node would be available as:

self.node.myComponent 

This would also work if there was only one.

if I need access to more than one of the JSComponents on the node:

self.node.components["MyComponent"]; // list of MyComponents on the node